client/app/lib/components/form/fields/DateTimePickerField.jsx

```diff
@@ -25,6 +25,11 @@ const styles = {
   dateTimeTextField: {
     marginRight: 5,
   },
+  dialogStyle: {
+    '.MuiDialog-paper': {
+      overflowY: 'visible',
+    },
+  },
```

Comment:
Instead of adding the tooltip CSS fix globally (which affected all dialogs and broke capybara tests), it's better to just leave it in the component that needs it.